pkgs/applications/emulators/86box/default.nix

```
@@ -48,38 +49,44 @@ stdenv.mkDerivation rec {
     ++ lib.optional (!enableDynarec) "-DDYNAREC=OFF"
     ++ lib.optional (!unfreeEnableDiscord) "-DDISCORD=OFF";
 
-  postInstall = lib.optional stdenv.isLinux ''
+  passthru = {
+    roms = fetchFromGitHub {
+      owner = "86Box";
+      repo = "roms";
+      rev = "v${finalAttrs.version}";
+      hash = "sha256-1HtoizO0QIGNjQTW0clzRp40h1ulw55+iTYz12UJSms=";
+    };
+  };
+  postInstall = (lib.optionalString stdenv.isLinux ''
```

Unnecessary `(` added, already pointed out by @SuperSandro2000 in [this comment](https://github.com/NixOS/nixpkgs/pull/288072/files#r1486462166) and in [my comment](https://github.com/NixOS/nixpkgs/pull/288072/files#r1533394386):

```suggestion
  postInstall = lib.optionalString stdenv.isLinux ''
```

---

Done